dolphinscheduler-registry/dolphinscheduler-registry-plugins/dolphinscheduler-registry-mysql/README.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+This module is the mysql registry plugin module, this plugin will use mysql as the registry center.
+
+# How to use
+
+If you want to set the registry center as mysql, you need to do the below two steps:
+
+1. Initialize the mysql table
```

Comment:
I think users need to add MySQL driver manually, configure the connection string, etc., if they are not already using MySQL as database.

Reply:
Yes, agree with you, I have added the datasource config, and remove the rely on dao module.

Reply:
> Yes, agree with you, I have added the datasource config, and remove the rely on dao module.

Hi, we still need to inform the users that they need to add MySQL driver manually as we can't bundle it in our distribution package